lib/mv_opentelemetry/broadway/messages.ex

```diff
@@ -0,0 +1,79 @@
+defmodule MvOpentelemetry.Broadway.Messages do
+  @moduledoc false
+
+  use MvOpentelemetry.SpanTracer,
+    name: :broadway,
+    events: [
+      [:broadway, :processor, :start],
+      [:broadway, :processor, :stop],
+      [:broadway, :processor, :message, :start],
+      [:broadway, :processor, :message, :stop],
+      [:broadway, :processor, :message, :exception]
+    ]
+
+  @spec handle_event([atom()], map(), map(), Access.t()) :: :ok
+  def handle_event([:broadway, :processor, :start], _measurements, meta, opts) do
+    attributes = [
+      {"broadway.topology_name", meta.topology_name |> inspect()},
+      {"broadway.stage", :processor},
+      {"broadway.index", meta.index},
+      {"broadway.processor_key", meta.processor_key},
+      {"broadway.messages_count", length(meta.messages)}
+    ]
+
+    event_name = "broadway.processor.start"
+    parent_context = OpentelemetryProcessPropagator.fetch_parent_ctx(1, :"$callers")
```

Comment:
If we are not inserting the context when we publish messages, I'd leave this part out, along with the links_from/1. I already have a plan to add it to Oban and we can revisit that later.